app/actors/hyrax/actors/file_set_actor.rb

```diff
@@ -110,12 +112,28 @@ def build_file_actor(relation)
           file_actor_class.new(file_set, relation, user)
         end
 
+        def io_decorator(file)
+          Hydra::Derivatives::IoDecorator.new(file, mime_for(file), label_for(file))
+        end
+
+        def mime_for(file)
+          if file.respond_to?(:content_type)
```

Comment:
Can you add documentation about which classes have which methods as you've done in `label_for`?